silence/silencepb/silence.proto

```diff
@@ -0,0 +1,57 @@
+syntax = "proto3";
+
+package silencepb;
+
+import "google/protobuf/timestamp.proto";
+
+// Matcher specifies a rule, which can match or set of labels or not.
+message Matcher {
+  // Type specifies how the given name and pattern are matched
+  // against a label set.
+  enum Type {
+    EQUAL = 0;
```

Comment:
Note that this makes `EQUAL` the default value (which might be your intention). If that's not intended, it is better to create an `INVALID` default value (or something like that). In the Prometheus `metrics.proto`, we (or Matt ;) missed the chance and made `COUNTER` the default (which is awkward).